operations/iscsi_sbps/iscsi_sbps.md

```diff
@@ -0,0 +1,242 @@
+# iSCSI SBPS (Scalable Boot Content Projection Service)
+
+iSCSI based boot content projection solution named Scalable Boot Content Projection Service (SBPS)
+is replacing current DVS (Cray Data Virtualization) to project boot content like `rootfs` and
+`programming environment` (PE) images. SBPS is an alternate projection capability for DVS aimed to offer better reliability, availability,
+security, ease and speed of deployment and ease of management than DVS. Key features of SBPS are as follows:
+
+1.	Provides open source friendly solution for read-only content projection (`rootfs` and `PE`) as it uses LIO (Linux IO) which is open source.
+2.	Horizontally scalable content projection service (iSCSI target side)
+3.	Delivers active/active IO operation from iSCSI initiator(s) to content projection service
+4.	Delivers seamless failover and failback for iSCSI initiator(s) on iSCSI target(s) or partial network failure
+5.	Supports projection over HSN (High Speed Network) and NMN (Node Management Network) 
+6.	Does not require additional hardware infrastructure (iSCSI target)
+7.	Co-exists with DVS until it is deprecated (Plan to deprecate DVS in CSM 1.7) 
+8.	Enables future work related to image access control, multi-tenancy, and related zero trust principles 
+9.	Does not require duplication of images from S3
+10.	Supports monitoring of content projection service for performance and reliability engineering
+11.	Aligns with future plans for similar functionality in next generation systems management solutions
+12.	Easy to deploy and manage. 
+
+# iSCSI SBPS solution details
+
+ ![iSCSI SBPS Architecture](../../img/iscsi-sbps-architecture.png)
+
+ Figure #1: Architecture diagram depicting read-only iSCSI based content projection (SBPS)
+
+As shown in figure #1, The basic configuration involves two iSCSI target/server (worker node) nodes
+and two iSCSI initiator/client (compute/UAN) nodes connected via HSN and/or NMN where I/O `multipathing`
+is configured. The `rootfs` and `PE` images are hosted in IMS and S3 respectively and both of these 
+images are mapped to “boot-images bucket of S3. DNS SRV and A [DNS - Domain Name Service, SRV – Service,

```

Comment:
Change `“boot-images` -> `"boot-images"`, that includes using simple double quotes instead of the complex double quote.
Change `SRV – Service` -> `SRV - Service`: a simple dash instead of the complex dash character.

Reply:
am using only the one dash key I have i.e '-', not sure why it is showing complex dash character.

Reply:
> Change `“boot-images` -> `"boot-images"`, that includes using simple double quotes instead of the complex double quote. Change `SRV – Service` -> `SRV - Service`: a simple dash instead of the complex dash character.

this is anyway changed to `boot-images` now.